configure

```diff
@@ -963,6 +1005,7 @@ if test $enable_cuda = yes ; then
     echo GDK_NVML_LIB_PATH=$gdk_nvml_lib_path >> $config
     echo CUB_PATH=$cub_path >> $config
     echo CUDNN_PATH=$cudnn_path >> $config
+    echo NCCL_PATH=$nccl_path >> $config
```

Comment:
Could you make that
[ -z "$nccl_path" ] || echo NCCL_PATH=$nccl_path >> $config
so that we don't output NCCL_PATH if nccl_path is not defined?